modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_folder.jsp

```diff
@@ -81,8 +81,8 @@ renderResponse.setTitle(dlEditFolderDisplayContext.getHeaderTitle());
 					</c:if>
 				</aui:fieldset>
 
-				<c:if test='<%= dlEditFolderDisplayContext.isFileEntryTypeSupported() && (dlEditFolderDisplayContext.hasFolderPermission(ActionKeys.ADVANCE_UPDATE) || !FeatureFlagManagerUtil.isEnabled(dlEditFolderDisplayContext.getCompanyId(), "LPD-42452")) %>'>
-					<aui:fieldset collapsed="<%= true %>" collapsible="<%= true %>" helpMessage="<%= dlEditFolderDisplayContext.getFileEntryTypeRestrictionsHelpMessage() %>" label="<%= dlEditFolderDisplayContext.getFileEntryTypeRestrictionsLabel() %>">
+				<c:if test="<%= dlEditFolderDisplayContext.isFileEntryTypeSupported() %>">
+					<aui:fieldset collapsed="<%= true %>" collapsible="<%= true %>" helpMessage="<%= dlEditFolderDisplayContext.getFileEntryTypeRestrictionsHelpMessage() %>" label="<%= dlEditFolderDisplayContext.getFileEntryTypeRestrictionsLabel() %>" style='<%= (dlEditFolderDisplayContext.hasFolderPermission(ActionKeys.ADVANCE_UPDATE) || !FeatureFlagManagerUtil.isEnabled(dlEditFolderDisplayContext.getCompanyId(), "LPD-42452")) ? "" : "display:none;" %>'>
```

Comment:
@gergelyszalay, I'm a bit concerned about this, because anyone would be able to change the HTML markup and interact with the fieldset